_data/api-swagger.yaml

```diff
@@ -649,6 +650,51 @@ definitions:
         }
       }
       ```
+
+  Itsme Identification:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      properties:
+        PhoneNumber:
+          type: string
+          readOnly: true
+          description:             
```

Comment:
Isn't this missing a pipe eg: `description: |` ?
Did you test this change locally?